src/components/SpruceForm/Widgets/LeafyGreenWidgets.tsx

```diff
@@ -38,19 +48,20 @@ export const LeafyGreenTextInput: React.VFC<
 }) => {
   const {
     ariaLabelledBy,
-    description,
     "data-cy": dataCy,
-    emptyValue,
+    description,
+    emptyValue = "",
     marginBottom,
     optional,
     warnings,
   } = options;
 
-  const hasError = !!rawErrors?.length;
+  const errors = filterInvisibleErrors(rawErrors);
+  const hasError = !!errors?.length;
 
   // Deduplicate errors due to a bug when using oneOf dependencies.
   // https://github.com/rjsf-team/react-jsonschema-form/issues/1590
-  const deduplicatedErrors = Array.from(new Set(rawErrors));
+  const deduplicatedErrors = Array.from(new Set(errors));
```

Comment:
What do you think of making one util function to process `rawErrors`? It seems like there's a chance that duplicate errors could be encountered in other widgets too so it seems like there's no harm in deduplicating processing all input types uniformly